src/main/java/org/broadinstitute/hellbender/cmdline/programgroups/ShortVariantDiscoveryProgramGroup.java

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.hellbender.cmdline.programgroups;
+
+import org.broadinstitute.barclay.argparser.CommandLineProgramGroup;
+import org.broadinstitute.hellbender.utils.help.HelpConstants;
+
+/**
+ * Tools that reassemble reads and call short variants. Tools also call non-reference probability, genotype and
```

Comment:
This description seems overly specific about internal mechanisms of the callers; would replace with e.g.:

Tools that perform variant calling and genotyping for short variants (SNPs, SNVs and Indels)

Reply:
Done.